docs/source/lts_selection.org

```diff
@@ -0,0 +1,151 @@
+#+TITLE: On selecting versions as LTS candidates
+#+AUTHOR: Aleksandr Petrosyan
+#+DATE: 12-08-2023
+* About
+
+In the interest of preserving the engineering resources, it is in our
+interests to limit the scope of post-release support. If one has
+several versions of Iroha running in parallel, they might need to
+replicate the support work as many times as there are parallel
+versions.
+
+Instead of attempting to support multiple versions badly, we try to
+cater to 80% of use cases with 20% of the effort and reach a
+compromise between supporting every released version equally and only
+supporting the latest stable version.
+
+Specifically, we support the latest =stable= release. Often this
+release is either not as well-tested, well-documented, or
+well-supported by the SDKs, in addition to often changing in terms of
+its API and ABI, resulting in a moving target for compatibility. For
+this reason we designate a specific version that is at least one of 12
```

Comment:
I guess we should refer to our monthly cadence right here, because 12 releases might sound confusing, alternatively we could just specify smth like "once a year"